types/bwip-js/tslint.json

```diff
@@ -5,6 +5,7 @@
         "no-consecutive-blank-lines": false,
         "no-padding": false,
         "no-unnecessary-type-assertion": false,
+        "no-outside-dependencies": false,
```

Comment:
Accidental/unnecessary change?

Reply:
I approved the `alttext?: string | undefined;` change, but recommend any changes to the linter config be done as a separate PR.

Reply:
Gah! Sorry my bad, I had to set this to run tests locally and didn't mean to push it. Fix coming.